python/cudf/tests/test_binops.py

```diff
@@ -70,7 +72,7 @@ def test_series_binop_scalar(nelem, binop, obj_class):
 
 @pytest.mark.parametrize('obj_class', ['Series', 'Index'])
 @pytest.mark.parametrize('cmpop', _cmpops)
-@pytest.mark.parametrize('dtype', ['int8', 'int16', 'int32', 'int64',
+@pytest.mark.parametrize('dtype', ['int8', 'int32', 'int64',
```

Comment:
Why did you remove `int16` from here?

Reply:
There was a test failure caused by the fact that we don't support `int16`. I mentioned [here](https://github.com/rapidsai/cudf/pull/992#issuecomment-466964664). I'll try again to see if it works.

Reply:
I believe we added a temporary typecast to `int32` and typecast back to `int16` to handle that temporarily, so we'll need to remove those if you haven't done so already.